doc/internal/man3/ossl_namemap_new.pod

```diff
@@ -64,6 +66,9 @@ ossl_namemap_name2num_n() does the same thing as
 ossl_namemap_name2num(), but takes a string length I<name_len> as well,
 allowing the caller to use a fragment of a string as a name.
 
+ossl_namemap_num2name() finds the name corresponding to the given
+I<number>.
```

Comment:
```suggestion
ossl_namemap_num2name() finds the I<idx>th name corresponding to the given
I<number>.
```

Reply:
One shouldn't have to go all the way to RETURN VALUES to find the documentation for *idx*th